### PR TITLE
Create lololo2.JSON

### PR DESCRIPTION
**Pull request by lololo2**
Adding a new address to the map

### CODE REVIEW

File: _pins/lololo2.JSON [hunk 1]
```diff
@@ -1,3 +1,3 @@
 githubHandle: lololo2
-latitude: 1
-longitude: 2
+latitude: 1111
+longitude: 2222
```
Comment (lololo2) [inline _pins/lololo2.JSON]: good

File: _pins/lololo2.JSON [hunk 2]
```diff
@@ -1,3 +1,3 @@
 githubHandle: lololo2
-latitude: 1
-longitude: 2
```
Comment (lololo2) [inline _pins/lololo2.JSON]: deleted

**Comment by lololo2** [comment]
Fixes #12619

File: _pins/lololo2.JSON [hunk 3]
```diff
@@ -0,0 +1,5 @@
+githubHandle: lololo2
+latitude: 1111
+longitude: 2222
+
+**bold text**
```
Comment (hectorsector) [inline _pins/lololo2.JSON]: @lololo2 you need `---` below and above your content. You should also remove the bold text and use real coordinates.

**Comment by mlarrubia** [comment]
cool

**Comment by PG7310** [comment]
Hi lololo2 change the file name from lololo2.JSON to lololo2.json.  (without capital letters in .json)  :smiley:

**Comment by crichID** [comment]
@lololo2 thank you for checking out our [on-demand classes](https://services.github.com/on-demand/). Since this file is not valid per the examples in the course, I am going to go ahead and close this Pull Request. If you would like to complete the activity, please feel free to re-open this Pull Request or start again. If you have any questions, please let us know!

**Pull request closed without merging**